R/rdeephaven/R/client_options_wrapper.R

```diff
@@ -61,16 +63,28 @@ ClientOptions <- R6Class("ClientOptions",
         #' @param auth_key Key to use for authentication, supplied as a string.
         #' @param auth_value Value to use for authentication, supplied as a string.
         set_custom_authentication = function(auth_key, auth_value) {
+            private$verify_string(auth_key, "auth_key")
+            private$verify_string(auth_value, "auth_value")
             self$internal_client_options$set_custom_authentication(auth_key, auth_value)
         },
 
         #' @description
         #' Set the session type of the console (e.g., "python", "groovy", etc.). The session type must be supported on the server.
         #' @param session_type Desired language of the console. "python", "groovy", etc.
         set_session_type = function(session_type) {
+            private$verify_string(session_type, "session_type")
             self$internal_client_options$set_session_type(session_type)
         },
 
         internal_client_options = NULL
+    ),
+    private = list(
+        verify_string = function(string_candidate, arg_name) {
```

Comment:
I would consider reversing the order of these args.  That would seem more natural.